mobile/src/main/java/org/openhab/habdroid/model/ServerProperties.java

```diff
@@ -119,6 +124,19 @@ public void onSuccess(String response, Headers headers) {
                     } catch (NumberFormatException nfe) {
                         // ignored: older versions without SSE support didn't return a number
                     }
+
+                    JSONArray jsonArray = result.optJSONArray("links");
```

Comment:
Nit: maybe name this variable `links` or something?

Reply:
`linksJsonArray`? I want to keep the fact that it's a json object in the name.

Reply:
Sounds fine.